cmd/juju/application/series_selector.go

```diff
@@ -116,3 +143,21 @@ func (s seriesSelector) userRequested(requestedSeries string) (string, error) {
 	logger.Infof(msgUserRequestedSeries, series)
 	return series, nil
 }
+
+func (s seriesSelector) validateSeries(seriesName string) error {
+	// if we're forcing then we don't need the following validation checks.
+	if s.force || len(s.supportedJujuSeries) == 0 {
```

Comment:
I am a bit confused about the second half of the check. So what happens if I request series 'foo' without `--force` and the charm does not define any supported series? Shouldn't this be an error?

Reply:
Yeah, i'll change that.